Jenkinsfile

```diff
@@ -81,6 +81,7 @@ def buildHive(args) {
   configFileProvider([configFile(fileId: 'artifactory', variable: 'SETTINGS')]) {
     withEnv(["MULTIPLIER=$params.MULTIPLIER","M_OPTS=$params.OPTS"]) {
       sh '''#!/bin/bash -e
+sw java 11
```

Comment:
@kgyrtkirk 
```
Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project hive-storage-api: Fatal error compiling: invalid target release: 11 -> [Help 1]
```
This didn't seem to work.

http://ci.hive.apache.org/blue/organizations/jenkins/hive-precommit/detail/PR-1241/5/pipeline